src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.ReflectedTypeData.cs

```diff
@@ -485,7 +485,7 @@ internal PropertyDescriptorCollection GetProperties()
             /// then a global Type.GetType is performed.
             /// </summary>
             [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
-                Justification = "typeName is annotated with DynamicallyAccessedMembers, which will preserve the type.")]
+                Justification = "Calling _type.Assembly.GetType on a non-assembly qualified type will still work. See https://github.com/mono/linker/issues/1895")]
```

Comment:
This is not 100% true, but I think it's probably good enough for now. Once we solver the linker issue referred here, we'll see if this needs some tweaking or not.